test/screenshot/lib/controller.js

```diff
@@ -252,17 +252,109 @@ class Controller {
       .then(
         (image) => {
           return new Promise((resolve, reject) => {
+            const {rows, cols} = this.getCropMatches_(image);
+            const cropRect = this.getCropRect_({rows, cols});
+            const {x, y, w, h} = cropRect;
+
             image
-              .autocrop()
+              .crop(x, y, w, h)
               .getBuffer(jimp.MIME_PNG, (err, buffer) => {
                 return err ? reject(err) : resolve(buffer);
-              });
+              })
+            ;
           });
         },
         (err) => Promise.reject(err)
       );
   }
 
+  getCropMatches_(image) {
```

Comment:
Add a function annotation here so it's clear that `image` is a Jimp image object